.gitignore

```diff
@@ -3,3 +3,4 @@ config
 tou
```

Comment:
This needs to be taken out. I believe Joseph made a pull request that took this out so you just need to merge those changes to this branch